R/TealReportCard.R

```diff
@@ -35,11 +35,7 @@ TealReportCard <- R6::R6Class( # nolint: object_name_linter.
     #'
     append_fs = function(fs) {
       checkmate::assert_list(fs)
-      super$append_text(yaml::as.yaml(fs, handlers = list(
-        POSIXct = function(x) format(x, "%Y-%m-%d"),
-        POSIXlt = function(x) format(x, "%Y-%m-%d"),
-        Date = function(x) format(x, "%Y-%m-%d")
-      )), "verbatim")
+      super$append_text(attr(fs, "formatted"), "verbatim")
       super$append_metadata("FS", fs)
```

Comment:
Do you actually need super here? I would have thought self$ would have worked unless this classes append_text and append_metadata has overridden the base class implementation and we specifically want the base class implementation

Reply:
updated.